person.rb

```diff
@@ -1,13 +1,14 @@
-require_relative 'nameable'
+class Person
+  attr_accessor :name, :age, :parent_permission, :rentals
```

Comment:
OPTIONAL
- Please remove the `parent_permission` attribute from the associations as it's not a specific column of a database table. It's just a custom boolean parameter that's set in the constructor of the class.